DEVELOPMENT.md

```diff
@@ -0,0 +1,90 @@
+# Development
+
+This doc explains how to setup a development environment so you can get started
+[contributing](https://www.knative.dev/contributing/) to `Knative Client`.
+Also take a look at:
+
+- [The pull request workflow](https://www.knative.dev/contributing/reviewing/)
+
+## Prerequisites
+
+Follow the instructions below to set up your development environment. Once you
+meet these requirements, you can make changes and
+[build your own version of Knative Client](#building-knative-client)!
+
+Before submitting a PR, see also [contribution guide](https://www.knative.dev/contributing/).
+
+### Sign up for GitHub
+
+Start by creating [a GitHub account](https://github.com/join), then setup
+[GitHub access via SSH](https://help.github.com/articles/connecting-to-github-with-ssh/).
+
+### Install requirements
+
+You must install these tools:
+
+1. [`go`](https://golang.org/doc/install): The language `Knative Client` is
+   built in (1.11.4 or later)
+1. [`git`](https://help.github.com/articles/set-up-git/): For source control
+1. [`kubectl`](https://kubernetes.io/docs/tasks/tools/install-kubectl/): For
+   managing development environments
+
+### Create a cluster
+
+1. [Set up a Knative](https://www.knative.dev/docs/install/)
+   - You can also setup using limited install guides for eg: Minikube/Minishift.
+
+### Setup your environment
+
+Let's setup local development envrionment.
```

Comment:
Is there more that should be in this section? If it's just the one sentence, perhaps we don't need this sentence or heading at all?

Reply:
Right.. removed a heading and sentence altogether.